astro/src/content/json/users/login-response.json

```diff
@@ -1,4 +1,6 @@
 {
+  "emailVerificationId": "YkQY5Gsyo4RlfmDciBGRmvfj3RmatUqrbjoIZ19fmw4",
```

Comment:
Verification Ids on sample responses are not a happy path thing, which seems to go against the grain of keep the samples as simple as possible, but I tried them out here. thoughts?

Reply:
If JSON only had comments. May be better to leave them out, how they appear in the response should be somewhat obvious from the response description.

Reply:
Done